src/main/java/com/flightstats/hub/metrics/CustomMetricsLifecycle.java

```diff
@@ -0,0 +1,29 @@
+package com.flightstats.hub.metrics;
+
+import com.google.common.util.concurrent.AbstractScheduledService;
+import com.google.inject.Inject;
+
+import java.util.concurrent.TimeUnit;
+
+public class CustomMetricsLifecycle extends AbstractScheduledService{
+    private CustomMetricsReporter collector;
```

Comment:
I like this much better than the nested Service classes we have everywhere else. 👍